### ISSUE TITLE
Saved solutions for all basic bonfires are missing from my profile

### ISSUE THREAD

**Issue by AdventureBear** [FreeCodeCamp/FreeCodeCamp#2562]
I'm sure they are still in the database but they are not being shown.  I'm sure there is something cool in the works to manage saved solutions, but I didn't see any other issues with this topic.

**Comment by AdventureBear** [comment]
Actually maybe it's a date t hing.   There are basic bonfires saved from June-ish when I did them for our local codecamp meeting.   I don't have any showing between Jan & June

**Comment by benschenker** [comment]
Hey @AdventureBear thanks for posting an issue!

Does #2066 have the same issue as you, or is it a different problem?

**Comment by AdventureBear** [comment]
No that's different, I'm talking about my codesolutions for bonfires.
Thanks!

On Sunday, August 23, 2015, Ben Schenker <notifications@github.com> wrote:

> Hey @AdventureBear <https://github.com/AdventureBear> thanks for posting
> an issue!
>
> Does #2066 <https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2066>
> have the same issue as you, or is it a different problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2562#issuecomment-133795073>
> .
>


-- 
Schedule a Swim lesson, Podcast or Coaching consult here:
http://www.booknow.so/SteelCity

**Comment by benschenker** [comment]
Any ideas on this @QuincyLarson?

**Comment by bugron** [comment]
@AdventureBear is this still an issue for you?

**Comment by AdventureBear** [comment]
Yes it is, not sure if this imag ewill show up but all my basic bonfires
between January and June ar missing, beetween "meet bonfire" and "make a
person".  I have a lot of good code I was "keeping" in there that I use for
reference or helping other folks.

-----
Suzanne Atkinson, MD
Clinical Instructor of Emergency Medicne
University of Pittsburgh School of Medicine
Emergency Resource Management, Inc.

Head Coach & Founder
Steel City Endurance Coaching

*Schedule a Swim lesson, Podcast or Coaching consult here:*
http://www.booknow.so/SteelCity

**Comment by izaac** [comment]
Got the same issue, I'm trying to review my past Bonfire solutions (Basic Algorithm Scripting ) and the challenges only show its initial state. Thanks.

**Comment by bugron** [comment]
@izaac the issue you're reporting is currently being discussed in #4857.

**Comment by AdventureBear** [comment]
anyone have any progress on this or ideas? I haven't got the codebase running locally and have not tried to solve it yet on my own. I'd really love to see all my old solutions because the WAy that i solved a problem helps me recall certain characteristics of problem solving

**Comment by legionista66** [comment]
I have the same issue now, Is there any way to fix it or all my  job is lost? How to prevent it in future ?

**Comment by SaintPeter** [comment]
@legionista66 
Looking at your profile: 
http://www.freecodecamp.com/legionista66

I see a bunch of solutions from November, then just one from January 20th.  Is this correct?

Are you using a different method to log in?  Could you possibly have a second account under a different login method?

**Comment by AdventureBear** [comment]
The "solution getter" codepen correctly pulls all of my solutions but they
are not reflected in my FCC profile. There's definitely a bug somewhere, I
have not made time to compare the two solutions.

On Monday, January 25, 2016, Rex Schrader <notifications@github.com> wrote:

> @legionista66 <https://github.com/legionista66>
> Looking at your profile:
> http://www.freecodecamp.com/legionista66
>
> I see a bunch of solutions from November, then just one from January 20th.
> Is this correct?
>
> Are you using a different method to log in? Could you possibly have a
> second account under a different login method?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2562#issuecomment-174604655>
> .
>


-- 
-----
Suzanne Atkinson, MD
Clinical Instructor of Emergency Medicne
University of Pittsburgh School of Medicine
Emergency Resource Management, Inc.

Head Coach & Founder
Steel City Endurance Coaching

*Schedule a Swim lesson, Podcast or Coaching consult here:*
http://www.booknow.so/SteelCity

**Comment by legionista66** [comment]
@SaintPeter 
I figure out how to get access to my solution but it is rather workaround and issue exist.
Now when I go to my portfolio and click on View Solution I can see it correctly but when I try to use "Map" and from it list chose the same solution (ticked as green ) I do not see my solution. 

In first case (from portfolio) link is correct and l looks like this: http://www.freecodecamp.com/challenges/Say%20Hello%20to%20HTML%20Elements?solution=%3Ch1%3EHello%20World%3C%2Fh1%3E%0A

For Map it looks like this: http://www.freecodecamp.com/challenges/say-hello-to-html-elements
and of course lead me to clear task instead showing me my solution. I remember that in last year Map was working correctly but now I notice you redesign it and now it cause this issue.

**Comment by ltegman** [comment]
@legionista66. This is expected behavior. Saved solutions from the DB are only accessible from your profile. Solutions loaded from the map are what is saved in localStorage as you work on a challenge and is intended to be temporary.
